### PR TITLE
Update docker compose to v3

### PR DESCRIPTION
**Pull request by chimame**
### Overview:概要
Update docker compose to version 3

### Technical changes:技術的変更点
* `volumes_from` is removed. So change to `volumes` option.
* Fix gems mounted volume so that it does not depend on host.  
This eliminates the need to recreate image when adding gem.

### Impact point:変更に関する影響箇所
Docker Engine:1.13.0+

Confirmation method
* [ ] `bin/docker setup` can be executed.
* [ ] `bin/docker s` can be executed.
* [ ] `http://localhost:4000` can open a top page.

### CODE REVIEW

**Comment by murajun1978** [comment]
docker v3 works `depends_on` ?

**Comment by chimame** [comment]
Thank you for reviewing my PR.
not write `depends_on` as removed in the [document](https://docs.docker.com/compose/compose-file/).
and also if you execute `bin/docker s`, no error is displayed.

**Comment by murajun1978** [comment]
yeah, `depends_on` still works.

but 👀  in the release notes like this.

> Containers dependencies can now be set up to wait on positive healthchecks
> when declared using depends_on. See the documentation for the updated
syntax.
> Note: This feature will not be ported to version 3 Compose files.

https://github.com/docker/compose/releases/tag/1.10.0

**Comment by chimame** [comment]
That's it.
>Healthcheck configuration can now be done in the service definition using
the healthcheck parameter

I think that I am writing about this. Is it wrong?

**Comment by murajun1978** [comment]
ah, Healthcheck only ?

i understanded it.

**Comment by murajun1978** [comment]
ok.

> depends_on still exists in v3, but healthcheck dependencies (and as a result, the extended syntax) will not be ported.

File: Dockerfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ RUN mkdir /app
 WORKDIR /app
 ADD Gemfile /app/Gemfile
 ADD Gemfile.lock /app/Gemfile.lock
-RUN bundle install -j4
+RUN bundle install --path vendor/bundle -j4
```
Comment (murajun1978) [inline Dockerfile]: Why did you add option ?
Are you using cached ?

Comment (chimame) [inline Dockerfile]: I want to keep dependent libraries in `volumes`. By default installed on system ruby.
So, we add `- /app/vendor` to` volumes` of `docker-compose.yml`.
Of course, this is still cached.

The greatest effect of this is to `bundle install` if a gem is added, but you do not have to recreate the docker image.
Because dependent libraries are persisted in `volumes`.

Comment (chimame) [inline Dockerfile]: Ah, if the dependency library can be persisted in the system's Ruby it is still fine.

Comment (murajun1978) [inline Dockerfile]: i see this option for development environment 👍

Comment (murajun1978) [inline Dockerfile]: Wrong ?

Comment (chimame) [inline Dockerfile]: > this option for development environment

That's right!
